src/main/java/com/EndavaTicketManagement/practica/repository/model/Customer.java

```diff
@@ -0,0 +1,20 @@
+package com.EndavaTicketManagement.practica.repository.model;
+
+
+public class Customer {
+    private String customerId;
+
+    public Customer(String customerId) {
+        this.customerId = customerId;
+        System.out.println("Creating Customer");
+    }
+
+    public String getCustomersId() {
```

Comment:
La ce ajuta clasele din **model**? vad ca mai ai un pachet **modelDBO** unde chiar ai pust Entity si alte tags pentru clasele care reprezinta tabelele din DB.

Daca clasele din modelDBO functioneaza si sunt ok, atunci as sterge pachetul **model** si pe cel de **modelDBO** l-as redenumi in **model**